app.py

```diff
@@ -94,6 +96,24 @@ def upload_file():
     )
 
 
+def list_files():
+    storage = Storage(client)
+    print_green("Running List Files API")
+    result = storage.list_files()
+    file_count = result['sum']
+    print("Total number of files {} ".format(file_count))
+    files = result['files']
+    print(files)
+
+
+def delete_file():
+    storage = Storage(client)
+    print_green("Running Delete File API")
+    result = storage.list_files()
+    first_file_id = result['files'][0]['$id']
```

Comment:
The console output for `delete_file` is kinda empty so it would be better if you could print out `result`.

Reply:
Also don't forget to update `README.md`

Reply:
Sure will do it

Reply:
Done, Updated READme and added console output for delete